src/@types/event.ts

```diff
@@ -94,6 +94,7 @@ export enum RelationType {
     Replace = "m.replace",
     Reference = "m.reference",
     Thread = "m.thread",
+    Reply = "m.in_reply_to",
```

Comment:
*technically* this isn't a RelationType, for extremely annoying reasons. Would advocate that we don't put it in this enum for now.

Reply:
TIL :upside_down_face: